cottoncandy/s3client.py

```diff
@@ -69,6 +69,11 @@ def __init__(self, bucket, access_key, secret_key, s3url, force_bucket_creation=
         self.bucket_name = None
 
         if bucket:
+            if not self.check_bucket_exists(bucket):
+                bucket_new = "glab-" + bucket.replace("_", "-")
+                if self.check_bucket_exists(bucket_new):
+                    bucket = bucket_new
+
```

Comment:
This is a bit hard-coded. Consider using the config value `mandatory_bucket_prefix` instead
https://github.com/gallantlab/cottoncandy/blob/main/cottoncandy/defaults.cfg#L11

Reply:
Oops, this is a personal hack, not meant to be in this pull-request. I will remove it.